docs/getting-started/advanced/content-styles.md

```diff
@@ -35,6 +35,10 @@ The content in the frontend of your application should now look the same as when
 	If you take a closer look at the content styles, you may notice they are prefixed with the `.ck-content` class selector. This narrows their scope when used in CKEditor&nbsp;5 so they do not affect the rest of the application. To use them in the front–end, **you will have to** add the `ck-content` CSS class to the container of your content. Otherwise, the styles will not be applied.
 </info-box>
 
+<info-box>
+	If you are not afraid to get your hands dirty, you can always create a custom CKEditor&nbsp;5 preset from the source code with **all** the CSS (both UI and the content) extracted to a separate file. See how to do that in a {@link getting-started/advanced/integrating-from-source-webpack#option-extracting-css dedicated guide}.
```

Comment:
IS the effect a single file, or didn't we have two style sheets at some point, one for the content and one for the UI?